exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -457,6 +464,9 @@
     .paragraph
       padding: 4px
 
+    .fixed-to-responsive
+      display: inline-block
+
```

Comment:
I'm concerned that we have so many custom styles targeting the private implementation details of Perseus. They're inevitably going to break and cause unexpected bugs when we upgrade Perseus (e.g. when they change the class names/meanings or HTML structure). We also have no way to test for that breakage, so it's likely that we'll release broken updates without knowing it.